app/assets/stylesheets/leaflet_overrides.css.less

```diff
@@ -1,3 +1,10 @@
 .leaflet-popup-content-wrapper, .leaflet-popup-tip {
   border: none;
 }
+.thumbnail{
+	background: #ffff!important; 
```

Comment:
Why not either 3 or 6 "f"s? Can we also get a space before !important, or look to sort the CSS structure so it's not needed?